### PR TITLE
Include Mission/Vision/History video from Chad Elish on frontpage

### PR DESCRIPTION
**Pull request by placer14**
Chad Elish put together a video which describes the NoM mission, vision, and history in an articulate video. Would this be worth including on the website?

The included video: https://www.youtube.com/watch?v=243QOUnOT38

This PR also includes a fix to the Board images URL. Please make certain these work with github hosted pages (which I'm pretty sure would be no problem).

Here's how the frontpage looks now:
https://www.dropbox.com/s/0hnclq8azp33h4r/nom_website_screenshot.png?dl=0

### CODE REVIEW

**Comment by danawoodman** [comment]
Thanks @placer14 I've passed this to the TG for vote!

File: index.html [hunk 1]
```diff
@@ -159,7 +160,7 @@ <h1 class='display-3'>Nation of Makers</h1>
           <div class='row'>
             <div class='col-lg-4'>
               <div class='card'>
-                <img class='card-img-top' src='/images/adam-savage.png' alt='Profile picture of Adam Savage'>
+                <img class='card-img-top' src='images/adam-savage.png' alt='Profile picture of Adam Savage'>
```
Comment (danawoodman) [inline index.html:163]: Any reason not to use absolute links here? Since the app is served by Github, these work and are a bit more explicit IMO

Comment (placer14) [reply index.html:163]: Because they break when loaded locally for development purposes.

**Comment by danawoodman** [comment]
Hey Mike, the consensus from the transition group is that we should wait until the Board of Directors has their first meeting and decides what to do with the website.  I'll keep this open so that we can not forget about it though! Thanks so much!  🤘😎

**Comment by stevegt** [comment]
If there are no objections, after the Apr 17th NWoM crunch is over I think we should set up a "videos" page, a youtube channel, or both, and include this one there.

**Pull request closed without merging**